nncf/torch/graph/transformations/commands.py

```diff
@@ -149,15 +149,32 @@ def union(self, other: "PTTransformationCommand") -> "PTTransformationCommand":
         raise NotImplementedError()
 
     def requires_graph_rebuild(self):
-        """
-        Return boolean flag to rebuild graph of model.
-
-        :return: Boolean flag.
-        """
         # Rebuild graph when adding quantization nodes.
         return self.priority == TransformationPriority.QUANTIZATION_PRIORITY
 
 
+class PTSharedFnInsertionCommand(PTTransformationCommand):
```

Comment:
Docstrings missing. When are you planning to move unified-scopes quantization to this new mechanism?

Reply:
Please see ticket 104304